howso/prediction_stats.amlg

```diff
@@ -37,6 +37,7 @@
 	;			two values = inclusive between
 	;   	- for nominal or string ordinal features:
 	;			n values = must match any of these values exactly
+	; holdout: flag, optional. If set to true, the cases specified in 'condition' will be held out entirely when making predictions for calculating stats.
```

Comment:
should probably add something like, "only used when condition is specified" or something along those lines